web/package.json

```diff
@@ -127,6 +128,7 @@
     "raf": "^3.4.1",
     "react-hot-loader": "^4.12.19",
     "redux-mock-store": "^1.5.3",
+    "selenium-webdriver": "^3.6.0",
```

Comment:
Is there a reason for selenium-webdriver instead of https://webdriver.io/ ?
In native we use the outdated wd dependency which I may want to replace.

Reply:
`webdriver.io` also wraps a testing framework. As we are already using jest for testing I didn't consider it. As for `selenium-webdriver` my main reasoning is based on the fact that it is managed by `Selenium` itself and should therefore be up-to-date for the most time. If there is a general consensus to use `webdriver.io` we can of course switch.